16.md

```diff
@@ -32,3 +32,18 @@ or
 ```
 
 The `text/email` entry MUST be used if the internet identifier corresponds to an actual email address.
+
+### Tags
+
+A `SERVICE` can offer multiple identifiers resolving to the same user by using the format `<username>+<tag>@<domainname>`.
+
+Upon seeing such an address, `WALLET` makes a GET request to `https://<domain>/.well-known/lnurlp/<username>+<tag>` endpoint.
+
+A `SERVICE` SHOULD strip the `+<tag>` part and continue as normal and MAY add a `text/tag` entry to the `metadata` JSON array:
+
+```
+[
+    "text/tag", // optional
+    string // the tag that has been passed in `<username>+<tag>`
```

Comment:
```suggestion
    string // the `<tag>` that has been passed in `<username>+<tag>`
```

nit: trying to avoid confusion on what this metadata field should include. If you quickly read it you might think it should be `<username>+<tag>`

Reply:
Good catch! Changed!  Thanks! :pray: